index.js

```diff
@@ -161,6 +212,17 @@ const line = []
 // console.log("%c----------", "color: red") 
 
 // ***** Scope & Closures - Question 2 *****
+function ticketNumberGeneratorFunc() {
+  let ticketNum = 0;
+  function takeATicketNumber (line) {
+  ticketNum += 1; 
+  line.push(ticketNum);
+  console.log(`Welcome. You are ticket number ${ticketNum}`) ;
+
+  }
+  return takeATicketNumber;
+}
+
```

Comment:
This is great! You could also return the function itself on like 217 so 223 wouldnt be needed, but it's not incorrect!